CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Change Log
 
+## 0.7.0 - 2021-05-12
+### Added
+- Project validation prior to `Create Static Web App...`
+- After creating a Static Web App, `git pull` is executed to retrieve workflow config
+- GitHub Configurations tree item when SWA is connected to local project
+- `Create Configuration File` command to add `staticwebapp.config.json`
+- Standard SKU for `Create Static Web App... (Advanced)`
```

Comment:
nit: I wanna make sure it doesn't sound like only the standard sku is supported in advanced create
```suggestion
- Standard SKU option for `Create Static Web App... (Advanced)`
```